### PR TITLE
Admin/invoice/us 36

### PR DESCRIPTION
**Pull request by chuckrenny**
36. Admin Invoice Show Page: Update Invoice Status

As an admin     
When I visit an admin invoice show page (/admin/invoices/:invoice_id)
I see the invoice status is a select field
And I see that the invoice's current status is selected
When I click this select field,
Then I can select a new status for the Invoice,
And next to the select field I see a button to "Update Invoice Status"
When I click this button
I am taken back to the admin invoice show page
And I see that my Invoice's status has now been updated

Created a select tag that updates the status attribute when the update button is clicked

### CODE REVIEW

File: app/controllers/admin/invoices_controller.rb [hunk 1]
```diff
@@ -6,4 +6,20 @@ def index
   def show
     @invoice = Invoice.find(params[:id])
   end
+
+  def update
```
Comment (kameronk92) [inline app/controllers/admin/invoices_controller.rb:10]: is there a reason to have two update methods here?

**Review by kameronk92 — CHANGES_REQUESTED** [review]

**Comment by kameronk92** [comment]
Really great work on this! Excited to see it in action. Just need to delete the extra `update` method on the `invoices_controller`

File: app/controllers/admin/invoices_controller.rb (hunk 1 above)
Comment (chuckrenny) [reply app/controllers/admin/invoices_controller.rb:10]: whoops, removed it

**Review by zkevkev — APPROVED** [review]
Nice work on this! Thanks for removing that extra update action.

**Pull request merged**